src/main/FixedDataKeyProvider.ts

```diff
@@ -0,0 +1,30 @@
+import {
+  crypto_kdf_KEYBYTES,
+  from_base64,
+  randombytes_buf,
+  ready,
+  to_base64
+} from "libsodium-wrappers"
+import { DataKey, DataKeyProvider } from "./DataKeyProvider"
+
+/** A DataKeyProvider that uses a single, fixed key. This is intended for testing  */
+export class FixedDataKeyProvider implements DataKeyProvider {
```

Comment:
It doesn't look like there's any material changes here outside of small changes necessary to implement the rename/tweaked `DataKeyProvider` interface.